src/views/Statistics/index.tsx

```diff
@@ -0,0 +1,240 @@
+import { Fragment } from 'react';
+import {
+  Anchor,
+  Badge,
+  Box,
+  Center,
+  Container,
+  Divider,
+  Flex,
+  Grid,
+  Group,
+  Image,
+  Paper,
+  ScrollArea,
+  Space,
+  Stack,
+  Text,
+  Title,
+  useMantineTheme,
+} from '@mantine/core';
+import { useMediaQuery } from '@mantine/hooks';
+import {
+  IconBuilding,
+  IconColorPicker,
+  // IconDownload,
+  IconExternalLink,
+  // IconEye,
+  IconFileFunction,
+  IconPlant,
+  IconSeeding,
+  IconTestPipe,
+} from '@tabler/icons';
+import { useLoaderData } from 'react-router-dom';
+import queries from '#/api/queries';
+
+import { Wave } from '#/components/Wave';
+import { Blob } from '#/components';
+
+import StatCard from './components/StatCard';
+import stats from '../../assets/stats.json';
+import ecologyEarth from '../../assets/ecology-earth.png';
+
+const recordStats = [
+  {
+    id: 'accessions',
+    name: 'Accessions',
+    icon: IconSeeding,
+  },
+  {
+    id: 'trials',
+    name: 'Trials',
+    icon: IconTestPipe,
+  },
+  {
+    id: 'treatments',
+    name: 'Treatments',
+    icon: IconColorPicker,
+  },
+];
+
+const speciesStats = [
+  {
+    id: 'speciesWithAccession',
+    name: 'Species with Accession',
+    icon: IconPlant,
+  },
+  {
+    id: 'speciesWithTrial',
+    name: 'Species with Trial',
+    icon: IconFileFunction,
+  },
+];
+
+interface Dataset {
+  datasetTitle: string;
+  datasetKey: string;
+}
+
+// eslint-disable-next-line import/prefer-default-export
+export function Component() {
+  const theme = useMantineTheme();
+  const mdOrLarger = useMediaQuery(`(min-width: ${theme.breakpoints.md})`, true);
+
+  const datasets = useLoaderData() as { [key: string]: Dataset };
+
+  // EPBC Calculations
+  const epbcDatasets = Object.entries(stats.epbcSpecies)
+    .sort(([, a], [, b]) => b - a)
+    .filter(([key]) => queries.DATA_RESOURCES.includes(key));
+
+  return (
+    <>
+      <Container size='lg' p='lg'>
+        <Space h={45} />
+        <Center>
+          <Text c='dimmed' size='lg' fw='bold'>
+            The following statistics are current as at October, 2024
```

Comment:
Should this date be hard-coded?

Reply:
Yeah - these statistics are manually calculated & sourced from a JSON file - these updates are only once per year, and the timeframe slightly differs for each year, the the text just needs to be changed each update.